charm/charmcraft.yaml

```diff
@@ -43,6 +43,8 @@ resources:
 provides:
   catalogue:
     interface: catalogue
+  provide-cmr-mesh:
```

Comment:
- Can we find a better name for the relation?
`provide-cmr-mesh` seems hard to parse for people who'd be inspecting the output of `juju status`.
- Please add a `description:` field with a brief explanation how this relation benefits catalogue. Something insightful beyond the trivial "Adds mesh capability".

Reply:
I imagine the name should be inspired by the difference between `interface: cross_model_mesh` and `interface: service_mesh`. It sounds the same to me.